src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -956,6 +981,20 @@ private static JsonSerializerOptions GetOrCreateDefaultOptionsInstance()
             return Interlocked.CompareExchange(ref s_defaultOptions, options, null) ?? options;
         }
 
+        [RequiresUnreferencedCode(JsonSerializer.SerializationUnreferencedCodeMessage)]
+        [RequiresDynamicCode(JsonSerializer.SerializationRequiresDynamicCodeMessage)]
+        private static JsonSerializerOptions GetOrCreateWebOptionsInstance()
+        {
+            var options = new JsonSerializerOptions
+            {
+                _propertyNameCaseInsensitive = true,
+                _jsonPropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+                _numberHandling = JsonNumberHandling.AllowReadingFromString
+            };
```

Comment:
Rather than open code the same set of options, can this just be `new JsonSerializerOptions(JsonSerializerDefaults.Web)`?

Reply:
This is also exposing the singleton as a mutable instance. Refer to the `Default` factory on how to make it read-only.